content/en/blog/_posts/2019-06-19-kubernetes-1-15-release-announcement.md

```diff
@@ -4,8 +4,9 @@ title: "Kubernetes 1.15: Extensibility and Continuous Improvement"
 date: 2019-06-19
 slug: kubernetes-1-15-release-announcement
 evergreen: true
+author: >
+  The 1.15 [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.15/release_team.md) 
```

Comment:
```suggestion
  [Kubernetes 1.15 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.15/release_team.md)
```
?

Reply:
Thanks for taking time to review. I have applied all the suggestion and review comments as part of the latest commit push.